From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Mon, 20 Dec 2021 17:24:15 +0100
Subject: [PATCH] {MAYBE UPS} drivers/gpu/drm/amd/display/dc/dcn10: mark maybe
 false-positive -Wunused-but-set-variable

With hardware interaction, you never know...
---
 drivers/gpu/drm/amd/display/dc/dcn10/dcn10_link_encoder.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/amd/display/dc/dcn10/dcn10_link_encoder.c b/drivers/gpu/drm/amd/display/dc/dcn10/dcn10_link_encoder.c
index f4b34c110eae..f8efe4af1094 100644
--- a/drivers/gpu/drm/amd/display/dc/dcn10/dcn10_link_encoder.c
+++ b/drivers/gpu/drm/amd/display/dc/dcn10/dcn10_link_encoder.c
@@ -1326,6 +1326,7 @@ void dcn10_link_encoder_update_mst_stream_allocation_table(
 		udelay(10);
 
 		value0 = REG_READ(DP_MSE_SAT_UPDATE);
+		(void)value0; /* Mark the variable as unused to please clang's -Wunused-but-set-variable */
 
 		REG_GET(DP_MSE_SAT_UPDATE,
 				DP_MSE_SAT_UPDATE, &value1);
-- 
